src/TeamSyncer.ts

```diff
@@ -469,7 +521,8 @@ export class TeamSyncer {
                 try {
                     await client.chat.postEphemeral({
                         user: channelItem.creator,
-                        text: `Hint: To bridge to Matrix, run the \`/invite @${user.name}\` command in this channel.`,
+                        text: `Please invite \`@${user.name}\` app to this channel (\`/invite @${user.name}\`) so that ` +
+                            `this channel is also available on Matrix side.`,
```

Comment:
I feel like we could improve the phrasing:

```suggestion
                        text: `Please invite \`@${user.name}\` to this channel (\`/invite @${user.name}\`) so that ` +
                            `the channel is also available on Matrix.`,
```